x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/filestructurefinder/TimestampFormatFinder.java

```diff
@@ -718,7 +793,11 @@ Boolean guessIsDayFirstFromMatches(TimestampFormat onlyConsiderFormat) {
 
             if (onlyConsiderFormat == null || onlyConsiderFormat.canMergeWith(match.timestampFormat)) {
 
+                // Valid indetermine day/month numbers will be in the range 1 to 31.
```

Comment:
typo: `indeterminate`; it is a confusing one to type :-)

Reply:
Gah, I fixed the exact same typo in another place in another recent commit